checkov/secrets/runner.py

```diff
@@ -225,6 +229,7 @@ def run(
                     root_folder=root_folder
                 ) or result
                 relative_file_path = f'/{os.path.relpath(secret.filename, root_folder)}'
+                # TODO add 'commit_hash' to the resource
```

Comment:
why do you want to add the commit hash to the resource, for tracking?

Reply:
yes, I wonder if we should separate the normal resource from the gis_history one?

Reply:
sounds like a good idea.